build.sbt

```diff
@@ -22,6 +16,14 @@ Global / onChangedBuildSource := ReloadOnSourceChanges
 val docMappingsApiDir = settingKey[String]("Subdirectory in site target directory for API docs")
 
 lazy val baseSettings = Seq(
+  // Finagle releases monthly using a {year}.{month}.{patch} version scheme.
+  // The combination of year and month is effectively a major version, because
+  // each monthly release often contains binary-incompatible changes.
+  // This means we should release at least monthly as well, when Finagle does,
+  // but in between those monthly releases, maintain binary compatibility.
+  // This is effectively PVP style versioning.
+  // We set this at the project-level instead of ThisBuild to circumvent sbt-typelevel checks.
+  versionScheme := Some("pvp"),
```

Comment:
This takes advantage of a loophole in sbt-typelevel that the versioning strategy is apparently only enforced at the `ThisBuild` level (although I might fix that 😅). So we can sneak it in as a project setting for now.

Btw, your custom strategy was not accepted by sbt when I moved the setting here.
```
java.lang.RuntimeException: unknown version scheme: year-month-patch
        at scala.sys.package$.error(package.scala:30)
        at sbt.internal.librarymanagement.VersionSchemes$.validateScheme(VersionSchemes.scala:36)
...
```

Reply:
Oh, I wondered if there was a list of schemes somewhere. This works for me.